Amazon.QLDB.Driver.Tests/QldbDriverTests.cs

```diff
@@ -231,19 +231,6 @@ public void TestExecuteWithActionAndRetryActionCanInvokeSuccessfully()
             Console.WriteLine);
         }
 
-        [TestMethod]
-        public void TestExecuteWithActionLambdaAndRetryPolicyCanInvokeSuccessfully()
-        {
-            var driver = new QldbDriver(
-                new SessionPool(() => Session.StartSession("ledgerName", mockClient.Object, NullLogger.Instance),
-                    QldbDriverBuilder.CreateDefaultRetryHandler(NullLogger.Instance), 4, NullLogger.Instance));
-
-            driver.Execute((txn) =>
-            {
-                txn.Execute("testStatement");
-            }, Driver.RetryPolicy.Builder().Build());
```

Comment:
This is different from other test. We shouldn't remove it.
This test is passing a `RetryPolicy`

Reply:
nvm, this is the same test as `TestExecuteWithActionAndRetryPolicyCanInvokeSuccessfully` (2 tests above)